pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1297,9 +1335,16 @@ func (cf *changeFrontier) Start(ctx context.Context) {
 		ctx = logtags.AddTag(ctx, "job", cf.spec.JobID)
 	}
 	ctx = logtags.AddTag(ctx, changeFrontierLogTag, nil /* value */)
+
```

Comment:
NB: PCR only traces the ingestion processor, where the vast majority of work occurs, and not its frontier processor. It may not be neccessary to trace the changefeed frontier processor, but you have more context.